### ISSUE TITLE
question - why the viewport is not set for the graph?

### ISSUE THREAD

**Issue by abonas** [kubernetes-ui/topology-graph#15]
I noticed that only the viewbox is set [here](https://github.com/kubernetes-ui/topology-graph/blob/master/topology-graph.js#L62)
but not the viewport (e.g. the width and the height).
wondering what is the reason for that?
(you can see info on the difference between viewport and viewbox [here](http://sarasoueidan.com/blog/svg-coordinate-systems/)

**Comment by abonas** [comment]
@stefwalter making sure you saw this

**Comment by stefwalter** [comment]
Because this breaks in various browsers. I would be happy for a sane patch that fixes the issue, and works cross browser.

**Comment by abonas** [comment]
> Because this breaks in various browsers. I would be happy for a sane patch that fixes the issue, and works cross browser.

can you please elaborate what breaks and where?

**Comment by stefwalter** [comment]
Not really, no it was too long ago. It was very hard to get support resizing of the resulting SVG across browsers. But I'm not against a pull request to fix this if there are issues.
